jupyter-notebooks/tasking-api/planet_tasking_api_order_creation.ipynb

```diff
@@ -0,0 +1,449 @@
+    "\n",
+    "* [`/order`](https://api.planet.com/tasking/v2/order/)\n",
+    "* [`https://api.planet.com/data/v1/quick-search`](https://api.planet.com/data/v1/quick-search)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Requirements\n",
+    "\n",
+    "#### Software & Modules\n",
+    "\n",
+    "This tutorial assumes familiarity with the [Python](https://python.org) programming language throughout. Familiarity with basic REST API concepts and usage is also assumed.\n",
+    "\n",
+    "We'll be using a **\"Jupyter Notebook\"** (aka Python Notebook) to run through the examples.\n",
+    "To learn more about and get started with using Jupyter, visit: [Jupyter](https://jupyter.org/) and [IPython](https://ipython.org/). \n",
+    "\n",
+    "For the best experience, download this notebook and run it on your system, and make sure to install the modules listed below first. You can also copy the examples' code to a separate Python files an run them directly with Python on your system if you prefer."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Basic Setup\n",
+    "\n",
+    "---\n",
+    "\n",
+    "Before interacting with the Planet Tasking API using Python, we will set up our environment with some useful modules and helper functions.\n",
+    "\n",
+    "* We'll configure *authentication* to the Planet Tasking API\n",
+    "* We'll use the `requests` Python module to make HTTP communication easier. \n",
+    "* We'll use the `json` Python module to help us work with JSON responses from the API.\n",
+    "* We'll use the `pytz` Python module to define the time frame for the order that we will be creating.\n",
+    "* We'll create a function called `p` that will print Python dictionaries nicely.\n",
+    "\n",
+    "Then we'll be ready to make our first call to the Planet Tasking API by hitting the base endpoint at `https://api.planet.com/tasking/v2`. \n",
+    "\n",
+    "Let's start by configuring authentication:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Authentication\n",
+    "\n",
+    "Authentication with the Planet Tasking API can be achieved using a valid Planet **API key**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can *export* your API Key as an environment variable on your system:\n",
+    "\n",
+    "`export PL_API_KEY=\"YOUR API KEY HERE\"`\n",
+    "\n",
+    "Or add the variable to your path, etc.\n",
+    "\n",
+    "To start our Python code, we'll setup an API Key variable from an environment variable to use with our requests:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import the os module in order to access environment variables\n",
+    "import os\n",
+    "\n",
+    "os.environ['PL_API_KEY']=input('Please provide your API Key')\n",
```

Comment:
Since any notebook repo users pass their api key in as an env var when running the docker container like this 
```
docker run -it --rm -p 8888:8888 -v $PWD:/home/jovyan/work -e PL_API_KEY=$PL_API_KEY planetlabs/notebooks
```
you could first check if the `PL_API_KEY ` env var and then fall back to the input if it's not already set.

Reply:
FWIW, I think this input pattern could be a nice enhancement to other notebooks in this repo, too.

Reply:
Yup, that's a good point. Change is below plus changing references from "order" to "tasking order"